conversion.md

```diff
@@ -0,0 +1,111 @@
+# Conversion to OCI Runtime Configuration
+
+When extracting an OCI Image into an [OCI Runtime bundle][oci-runtime-bundle], two orthogonal components of the extraction are relevant:
+
+1. Extraction of the root filesystem from the set of [filesystem layers](layers.md).
```

Comment:
> This document only concerns tools that implement this conversion (that intend to actually also implement either a runtime or to produce a config.json to the user).

In practice, this conversion will be performed by systems that will make this opaque. The advice here applies, but may not be verifiable without an introspection mechanism (read: leaky abstraction). For example, we'd like to be able to pass this spec through several systems before writing out the resulting config.json.